coding_introduction/CodingIntroduction.md

```diff
@@ -308,6 +373,33 @@ Regular files have filemode 644.
 - use `git add *`, **NEVER**.
 
 
+### Best Practices: Pull Requests
+For Pull Requests, there are two kind of best practices, related to the Pull Request itself, and to the Pull Request
+Review, that we established in [@ipa320](https://github.com/ipa320).
+
+- Pull Request
+  - keep the PR small (<100 changed files, <1000 changed lines of code)
```

Comment:
for a normal feature PR <300 LOC and <10 Files should be more than enough.

I agree to have exceptions for (i) new packages and (ii) parameter adaptions. But I'd like to still specify a lower limit so that people star thinking about if this is really necessary. (If we all agree, I'd like to add this checks to the Travis tests some time...)